### PR TITLE
Privacy button alignment made vertical

### PR DESCRIPTION
**Pull request by talkshrey**
* Resolves https://github.com/wagtail/wagtail/issues/8190

### CODE REVIEW

**Comment by lb-** [comment]
@talkshrey thanks so much for taking the time to contribute, I have not looked at this solution myself but I see there are some comments from @zerolab  about this on https://github.com/wagtail/wagtail/issues/8173

Please give a go at adding some tests and if you get the chance can you update the description with the solution you have implemented, plus some info on how to test this out.

**Comment by gasman** [comment]
Thanks @talkshrey!
Given that this issue is presumably not specific to these two views, and is likely to occur in any place checkboxes / radio buttons are used (including page edit forms built by site implementors), I'm wondering if this is something we can fix globally instead?

**Comment by talkshrey** [comment]
> Thanks @talkshrey!
> Given that this issue is presumably not specific to these two views, and is likely to occur in any place checkboxes / radio buttons are used (including page edit forms built by site implementors), I'm wondering if this is something we can fix globally instead?

I had the same thought process but I am not able to find dynamic classes which were created in django

File: docs/contributing/developing.rst [hunk 1]
```diff
@@ -323,7 +323,7 @@ To start this simple server, run the following commands:
 .. code-block:: console
 
     $ cd docs/_build/html/
-    $ python -mhttp.server 8080
+    $ python -m http.server 8080
```
Comment (lb-) [inline docs/contributing/developing.rst:326]: @talkshrey  please can you put this change up as a separate PR

**Comment by lb-** [comment]
@talkshrey  it looks like your changes have been lost, may I recommend you create a new PR that uses a new branch created from `main` with just the changes you need for this resolution.

**Comment by talkshrey** [comment]
> @talkshrey  it looks like your changes have been lost, may I recommend you create a new PR that uses a new branch created from `main` with just the changes you need for this resolution.
> 

Yeah I'll try again

**Comment by gasman** [comment]
Closing for now as none of the original commits remain at this point - feel free to open a new PR.
